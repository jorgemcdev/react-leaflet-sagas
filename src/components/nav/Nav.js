import React from 'react';
import { Link } from 'react-router-dom';

const Nav = () => (
  <div style={{ margin: 10 }}>
    <div className="ui secondary menu">
      <Link to="/" className="item">Home</Link>
      <div className="right menu" />
    </div>
  </div>
);

export default Nav;
