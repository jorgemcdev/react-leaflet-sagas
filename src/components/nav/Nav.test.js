/* eslint no-undef: 0 */
import React from 'react';
import renderer from 'react-test-renderer';
import { MemoryRouter } from 'react-router';
import Nav from './Nav';

test('Renders Navigation Header', () => {
  const component = renderer.create(
    <MemoryRouter>
      <Nav />
    </MemoryRouter>,
  );
  expect(component).toMatchSnapshot();
});
