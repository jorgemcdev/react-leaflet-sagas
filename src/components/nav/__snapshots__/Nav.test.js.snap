// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders Navigation Header 1`] = `
<div
  style={
    Object {
      "margin": 10,
    }
  }
>
  <div
    className="ui secondary menu"
  >
    <a
      className="item"
      href="/"
      onClick={[Function]}
    >
      Home
    </a>
    <div
      className="right menu"
    />
  </div>
</div>
`;
